packages/xo-common/src/api-errors.js

```diff
@@ -200,3 +189,13 @@ export const notEnoughResources = create(24, data => ({
   data, // [{ resourceSet, resourceType, available, requested }]
   message: 'not enough resources in resource set',
 }))
+
+export const incorrectState = create(25, ({ actual, context, expected, ids }) => ({
+  data: {
+    actual,
+    context,
+    expected,
+    ids,
```

Comment:
Not generic.

Reply:
Please give me what you want as result, suggestion will be better. Thank you :)